### ISSUE TITLE
Runn a specific rule from the cli

### ISSUE THREAD

**Issue by polaroidkidd** [IanVS/eslint-nibble#95]
Hi

1. Loving this tool! It's saving me from a lot of head-aches.
2. I've looked through the readme, but I'm not sure I understand the `--non-interactive` flag correctly. Does this mode allow me to specify a rule in the cli which nibble would auto-fix?

I have a monorepo with 28 apps in it and navigating into every app one-at-a-time is a bit of pain. I've tried using the rule in combilation with the `--rule` flag but the desired outcome isn't quite there yet.

**Comment by IanVS** [comment]
@polaroidkidd Yes you should be able to specify something like `eslint-nibble . --non-interactive --rule semi` In order to run eslint against your code, using your project config, but only for the "semi" rule.  If that's not working, can you share the full command that you are running?

**Comment by IanVS** [comment]
Are you trying to automatically fix a particular rule?  If so, https://github.com/IanVS/eslint-filtered-fix might be what you're looking for (it's what eslint-nibble uses under-the-hood).

**Comment by IanVS** [comment]
Great, I hope that meets your needs, and let me know if you hit any other issues.  Since you have such a large project, I'll point out I recently became aware of that https://github.com/mizdra/eslint-interactive, which it sounds like might have better performance than eslint-nibble (though maybe not eslint-filtered-fix).  If you try that out, I'd be curious to hear your thoughts on it.

**Comment by mizdra** [comment]
eslint-interactive has a feature called [Programmable API](https://github.com/mizdra/eslint-interactive/blob/627f1c5e6b6b983cdabe4e945db5619aa6246fe0/docs/programmable-api.md). You can probably use it to do the same thing as eslint-filtered-fix (more flexible than eslint-filtered-fix, but the API is not as easy). If you need it, please see below.

- [Documentation](https://github.com/mizdra/eslint-interactive/blob/627f1c5e6b6b983cdabe4e945db5619aa6246fe0/docs/programmable-api.md)
- [Available API](https://github.com/mizdra/eslint-interactive/blob/627f1c5e6b6b983cdabe4e945db5619aa6246fe0/src/index.ts)
  - [`Core` class](https://github.com/mizdra/eslint-interactive/blob/627f1c5e6b6b983cdabe4e945db5619aa6246fe0/src/core.ts#L75-L240)

**Comment by polaroidkidd** [comment]
oh, sweet! I passed this on to the team developing our internal react-scripts version as this tool greatly helps migrating the code base I'm managing to the new eslint rules.
